mc.yaml

```diff
@@ -73,10 +73,22 @@ subpackages:
           # Restore path
           find ${{targets.subpkgdir}}/opt/bitnami -type f -exec sed 's#${{targets.subpkgdir}}##g' -i {} \;
     test:
+      environment:
+        contents:
+          packages:
```

Comment:
These where previously defined at the image level as runtime dependencies - moving to package level for consistency with other similar packages.